client/webserver/site/src/js/orderutil.ts

```diff
@@ -135,41 +137,41 @@ export function matchStatusString (m: Match) {
     // When revoked, match status is less important than pending action if still
     // active, or the outcome if inactive.
     if (m.active) {
-      if (m.redeem) return 'Revoked - Redemption Sent' // must require confirmation if active
+      if (m.redeem) return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.prep(intl.ID_MATCH_STATUS_REDEMPTION_SENT) }) // must require confirmation if active
       // If maker and we have not redeemed, waiting to refund, assuming it's not
       // revoked while waiting for confs on an unspent/unexpired taker swap.
-      if (m.side === Maker) return 'Revoked - Refund PENDING'
+      if (m.side === Maker) return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.prep(intl.ID_MATCH_STATUS_REFUND_PENDING) })
       // As taker, resolution depends on maker's actions while waiting to refund.
-      if (m.counterRedeem) return 'Revoked - Redeem PENDING' // this should be very brief if we see the maker's redeem
-      return 'Revoked - Refund PENDING' // may switch to redeem if maker redeems on the sly
+      if (m.counterRedeem) return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.prep(intl.ID_MATCH_STATUS_REDEEM_PENDING) }) // this should be very brief if we see the maker's redeem
+      return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.prep(intl.ID_MATCH_STATUS_REFUND_PENDING) }) // may switch to redeem if maker redeems on the sly
     }
     if (m.refund) {
-      return 'Revoked - Refunded'
+      return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.prep(intl.ID_MATCH_STATUS_REFUNDED) })
     }
     if (m.redeem) {
-      return 'Revoked - Redemption Confirmed'
+      return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.prep(intl.ID_MATCH_STATUS_REDEMPTION_CONFIRMED) })
     }
-    return 'Revoked - Complete' // i.e. we sent no swap
+    return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.prep(intl.ID_MATCH_STATUS_COMPLETE) }) // i.e. we sent no swap
   }
 
   switch (m.status) {
     case NewlyMatched:
-      return 'Newly Matched'
+      return intl.prep(intl.ID_MATCH_STATUS_NEWLY_MATCHED)
     case MakerSwapCast:
-      return 'Maker Swap Sent'
+      return intl.prep(intl.ID_MATCH_STATUS_MAKER_SWAP_CAST)
     case TakerSwapCast:
-      return 'Taker Swap Sent'
+      return intl.prep(intl.ID_MATCH_STATUS_TAKER_SWAP_CAST)
     case MakerRedeemed:
       if (m.side === Maker) {
-        return 'Redemption Sent'
+        return intl.prep(intl.ID_MATCH_STATUS_REDEMPTION_SENT)
       }
-      return 'Maker Redeemed'
+      return intl.prep(intl.ID_MATCH_STATUS_MAKER_REDEEMED)
     case MatchComplete:
-      return 'Redemption Sent'
+      return intl.prep(intl.ID_MATCH_STATUS_REDEMPTION_SENT)
     case MatchConfirmed:
-      return 'Redemption Confirmed'
+      return intl.prep(intl.ID_MATCH_STATUS_REDEMPTION_CONFIRMED)
   }
-  return 'Unknown Match Status'
```

Comment:
Some thoughts about this one.
- This is in a Match card, and there's "Status" heading.  I think this can just be "Unknown"
- There's an existing `ID_UNKNOWN` that is simply `'unknown'`
- we've never seen this happen, which is a good sign, no egregious bugs with the status values

All considered, I don't think we need yet another string to translate.

![image](https://user-images.githubusercontent.com/9373513/205954684-8439456e-9873-4ebd-8355-488265addc1f.png) :heavy_check_mark: 

vs

![image](https://user-images.githubusercontent.com/9373513/205954445-cd0be3e7-4a34-4ccd-a954-629ba27ac7c9.png) :x: